test/integration_tests/rnnt_pipeline_test.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+import torchaudio
+from torchaudio.prototype.pipelines import EMFORMER_RNNT_BASE_LIBRISPEECH
+
+
+@pytest.mark.parametrize(
+    "bundle,lang,expected",
+    [
+        (
+            EMFORMER_RNNT_BASE_LIBRISPEECH,
+            "en",
+            # SentencePiece token ids for "i have that curiosity beside me at this moment"
+            [4096, 18, 135, 60, 3526, 1931, 113, 100, 138, 619],
```

Comment:
Why not test on the final output from sentence piece?

Reply:
i chose to test on just the raw ids because we haven't made `sentencepiece` an explicit dependency

Reply:
I don't have a strong preference between the two, but one option could to have `sentencepiece` be an optional testing dependency like the ones listed [here](https://github.com/pytorch/audio/blob/main/CONTRIBUTING.md#install-torchaudio)